### PR TITLE
Task/rbac continue

### PR DESCRIPTION
**Pull request by EduardoVaca**
#### What's this PR do?
Finishes the RBAC

#### Where should the reviewer start?
Login with different users with different roles

#### How should this be manually tested?
Login with different users with different roles

#### Any background context you want to provide?
Create users with different roles

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: src/app/general-user/works/works.spec.ts [hunk 1]
```diff
@@ -135,6 +137,7 @@ describe('WorksComponent tests', () => {
                   },
                   deps: [MockBackend, BaseRequestOptions]
                 },
+                { provide: ActivatedRoute, useClass: class { navigate = jasmine.createSpy("navigate"); } },
```
Comment (marccolm) [inline src/app/general-user/works/works.spec.ts]: Change "" to ''

File: src/app/general-user/works/works.spec.ts [hunk 2]
```diff
@@ -150,10 +153,14 @@ describe('WorksComponent tests', () => {
 
         // WorksComponent test instance.
         component = fixtureParent.componentInstance;
+        component.currentUser = new CurrentUser({ id: 1, username: 'hi'});
+        console.log(component.currentUser);
```
Comment (JuanGro) [inline src/app/general-user/works/works.spec.ts]: hide console log please

File: src/app/general-user/works/works.component.ts [hunk 3]
```diff
@@ -372,18 +399,11 @@ export class WorksComponent implements OnInit, OnChanges {
   };
 
   /**
-  * Reloads user list after
+  * Reloads user list after receiving notifiations.
```
Comment (JuanGro) [inline src/app/general-user/works/works.component.ts]: notifications*

File: npm-debug.log [hunk 4]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (JuanGro) [inline npm-debug.log]: Remove npm-debug.log please

**Comment by JuanGro** [comment]
And follow the standard code please.

screen shot 2017-04-26 at 1 54 58 am

File: npm-debug.log (hunk 4 above)
Comment (EduardoVaca) [inline npm-debug.log]: DONE

File: src/app/general-user/works/works.spec.ts (hunk 2 above)
Comment (EduardoVaca) [inline src/app/general-user/works/works.spec.ts]: done

**Pull request merged**